### ISSUE TITLE
Translate questions and recommendations

### ISSUE THREAD

**Issue by gcharest** [sara-sabr/ITStrategy#1161]
Need to have all questions properly available in both official languages before pushing online.

**Comment by myp192** [comment]
No action until #1191

**Comment by smellems** [comment]
With translation for editing and French translation.  Should have it back by 2021-02-25 12:00
